config/locales/en/nsm/additional_fess.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
`additional_fess.yml` @elken you have something you want to tell us 😄 ?

Reply:
Not sure, could be a rebase gone awry

Reply:
good catch!